creedengopython.properties

```diff
@@ -1,12 +1,20 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Python language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/creedengo-python
-publicVersions=2.0.1
-archivedVersions=2.0.0
+publicVersions=2.1.0
+archivedVersions=2.0.0,2.0.1
 
 defaults.mavenGroupId=org.green-code-initiative
 defaults.mavenArtifactId=creedengo-python-plugin
 
+2.1.0.description=add 5 new rules; compatibility with SonarQube 25.5.0; upgrade Integration Test System; update jdk 11 to 17
+2.1.0.sqcb=[24.12,LATEST]
+2.1.0.sqs=[10.8,LATEST]
+2.1.0.sqVersions=[9.9,10.7.*]
+2.1.0.date=2025-08-02
+2.1.0.downloadUrl=https://github.com/green-code-initiative/creedengo-python/releases/download/2.1.0/creedengo-python-plugin-2.1.0.jar
+2.1.0.changelogUrl=https://github.com/green-code-initiative/creedengo-python/releases/tag/2.1.0
+
 2.0.1.description=improve library versions; check and upgrades for SonarQube until 25.3.0 version
 2.0.1.sqcb=[24.12,LATEST]
```

Comment:
Only one version at a time can be compatible with `LATEST` and no archived versions can be compatible with `LATEST`. When you archive a version the expected behavior is to edit `LATEST` out of its end ranges and replace it with the current version(s) - unless you know of a specific incompatibility that would cause you to use an earlier version as the new end of the range.